integration-tests/features/autosuggest.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Autosuggest
+
+    Background:
+        * url apiHost
+
+    Scenario: we need to confirm that the feature is available
+        Given path 'Autosuggest', 'status'
+        When method get
+        Then status 200
+        And match response == 'alive!'
+
+    Scenario Outline: we want valid terms most closely matching what the user has typed so far
+        Given path 'Autosuggest', 'cgov', lang, term
```

Comment:
When tests fail,  Karate doesn't tell us the inputs. To fix this, on the line after the `Scenario Outline`, please add a line listing the inputs.  So this would become

```gherkin
    Scenario Outline: we want valid terms most closely matching what the user has typed so far
        for lang: <lang>, term: <term>, size: <size>

        Given path 'Autosuggest', 'cgov', lang, term
```

This change is needed for all Scenario outlines in all `.feature` files.

Reply:
Extra lines added.